{
  try {
    PyPullUpPresenter sut=configureByClass(className);
    myMocksControl.replay();
    sut.launch();
  }
 catch (  IllegalArgumentException ignored) {
    return;
  }
  Assert.fail("Presenter should throw exception, but it returned list of parents instead: " + myParentsCapture.getValue());
}
